### ISSUE TITLE
Unable to connect Web sdk user to native Sdk user  

### ISSUE THREAD

**Issue by iamsb1896** [AgoraIO-Community/AgoraWebSDK-NG#146]
### How to connect native client to web client ?
Hi, We are trying to build a multiplatform system using Agora-SDK. We are able to see the client joined from web side to native but Native Client is visible on Web ?

Native Platform : Android (Flutter)
Web Browser: Chrome

Web Client

    var client = AgoraRTC.createClient({ mode: "live", codec: "h264" });

 Native   
 
    _engine = await RtcEngine.create(appId);
    await _engine.enableWebSdkInteroperability(true);

**Comment by plutoless** [comment]
it should just be working. are you sure both side have successfully joined the channel?

**Comment by iamsb1896** [comment]
Yes, It's working I am able to connect Native Client to Native Client or Web Client to web client.

I am able to display Remote web client(user) on native side using "userJoined" event listeners whenever web client joins the session. But, 

    client.on("user-published", handleUserPublished); 

is not getting trigerred when Native User joins the session.

Web SDK:

AgoraRTC_N

    <script src="https://download.agora.io/sdk/release/AgoraRTC_N.js"></script>


Flutter SDK:

     agora_rtc_engine: ^3.2.1
